### PR TITLE
Worker Apps in AppLens

### PR DESCRIPTION
**Pull request by SharmaAjay19**
This PR achieves three things:
**1. Observer integration in AppLens for Worker Apps.
2. Passing Geomaster information to diagnostics backend for Geomaster data provider for worker apps.
3. Worker apps search experience on the AppLens main landing page**

**Landing Page Experience**

User selects Worker App from dropdown
image

User Enters Worker App name and clicks Go
image

Detector view with data confirmation of Geomaster data and observer data:
image

### CODE REVIEW

File: AngularApp/projects/applens/src/app/modules/dashboard/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -128,6 +128,12 @@ export class DashboardComponent implements OnDestroy {
                 this.resourceService.imgSrc = this.resourceService.altIcons['Xenon'];
             }
         }
+        else if (serviceInputs.resourceType.toString() === 'Microsoft.Web/workerApps')
```
Comment (cindylovescoding) [inline AngularApp/projects/applens/src/app/modules/dashboard/dashboard/dashboard.component.ts:131]: Can we use lowercase match just to be safe?

File: AngularApp/projects/applens/src/assets/enabledResourceTypes.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
             "searchSuffix": "Azure Logic App"
         },
         {
-            "service": "ResourceService",
+            "service": "WorkerAppService",
             "resourceType": "Microsoft.Web/workerApps",
             "templateFileName": "WorkerApp",
             "imgSrc": "assets/img/Azure-KubernetesService-Logo.png",
```
Comment (cindylovescoding) [inline AngularApp/projects/applens/src/assets/enabledResourceTypes.json:45]: Is this gonna use AKS logo?

File: AngularApp/projects/applens/src/app/modules/dashboard/dashboard/dashboard.component.ts (hunk 1 above)
Comment (SharmaAjay19) [reply AngularApp/projects/applens/src/app/modules/dashboard/dashboard/dashboard.component.ts:131]: Nice catch, actually even other resources conditions are messed up, will fix all of them.

File: AngularApp/projects/applens/src/assets/enabledResourceTypes.json (hunk 2 above)
Comment (SharmaAjay19) [reply AngularApp/projects/applens/src/assets/enabledResourceTypes.json:45]: Yes for now, until they get a logo.

**Review by XiaoxuHU — APPROVED** [review]

File: AngularApp/projects/applens/src/assets/enabledResourceTypes.json (hunk 2 above)
Comment (michimune) [reply AngularApp/projects/applens/src/assets/enabledResourceTypes.json:45]: The logo on the portal looks like this:
image

**Review by cindylovescoding — APPROVED** [review]

**Pull request merged**